Resources/en.lproj/Localizable.strings

```diff
@@ -1567,6 +1567,14 @@
 "product_title_wiki" = "Wikipedia";
 "product_desc_wiki" = "Wikipedia data";
 "product_desc_ext_wiki" = "Get information about points of interest from Wikipedia. It is your pocket offline guide - just enable Wikipedia plugin and enjoy articles about objects around you.";
+"plugin_wikipedia_description" = "Get info about points of interest from Wikipedia, a pocket offline guide featuring articles about places and destinations.";
```

Comment:
What is the point of this line? It's completely identical to ```"product_desc_ext_wiki"``` but with poor grammar.